### DISCUSSION TITLE
Reducing the cost of producing small batches

### DISCUSSION THREAD

**Discussion by ddv2005** [revk/xESP32-Faikout#65] [Ideas]
Hello, 

I just installed Daiken AC with 4 indoor units and was shocked by WIFI modules prices. And your project has been a lifesaver for me. But I am in USA and no way to buy your boards on Amazon. I tried to order on jlcpcb by your production files and found that 75% of price is additional fees because of non Economic assembly on both sides. I made your design in EasyEda but put everything on the one side. Also ESP32 module required Standard assembly too and I replaced it to generic D1 Mini ESP32 board with ESP32-WROOM-32 module on board.  As result I got 39.86 mm* 33.16 mm board that cost just $36 for 5 assembled  boards (vs $144 for original design) and needs $6 D1 Mini ESP32 module + JST angled connector. Also it has build-in USB port and you can program it directly without any additional hardware. I guess that if you make additional design as SHIELD for existing ESP32 boards then it would be great help to people who wants to build boards by itself for cheap.

![image](https://github.com/revk/ESP32-Faikin/assets/1864211/45d61147-aac6-4a68-b703-0bc8b002e3a3)

**Comment by revk** [comment]
I can certainly look at an alternative design...

I know some people have managed to get shipping from Amazon using that parcel forwarding company, if that helps.

**Comment by revk** [comment]
I am thinking that for most people, they will just buy some standard ESP32 module and wire it up, rather than ordering.

That said, I am constantly battling JLCPCB on what they do and what they charge, so it may end up done differently over time.

Thanks for the feedback.
